src/devtools/views/Components/Tree.css

```diff
@@ -24,10 +24,6 @@
   font-family: var(--font-family-monospace);
   font-size: var(--font-size-monospace-normal);
   line-height: var(--line-height-data);
-  border: 0.25rem solid transparent;
-}
-.List:focus-within {
-  border-color: var(--color-button-background-focus);
```

Comment:
Just revert the border/padding change so we won't have the border offers l offset bug either?

Reply:
Maybe I misunderstood — but isn't this the fix for it? I thought I fixed it by removing borders (like in this PR). Or are you saying you want to add white paddings back? (They don't seem to influence that issue.)